<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensitivityAnalysis</class>
 <widget class="QDialog" name="SensitivityAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="wg_drawWindow" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>461</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 227, 227);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_selectVar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>孔隙度Φ(f)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>气井射孔段底部高度h2(m)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>气井打开厚度hp(m)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>表皮系数S(f)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>供给半径re(m)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>半球面向心流半径r1(m)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>气井射孔段底部产量q(10^4*m^3/d)</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="bt_startSens">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>进行分析</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sensRange">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>分析范围</string>
   </property>
  </widget>
  <widget class="QLabel" name="label__">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sensMin">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sensMax">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tv_resultTable">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>231</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_saveGraph">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_copyTableData">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制表格</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
